lib/openstudio/extension.rb

```diff
@@ -41,6 +41,24 @@ module Extension
     class Extension
       attr_accessor :root_dir
 
+      # max number of datapoints to run
+      MAX_DATAPOINTS = Float::INFINITY
+
+      # number of parallel jobs
+      NUM_PARALLEL = 7
+
+      # do simulations
+      DO_SIMULATIONS = false
+
+      # specify to run the baseline simulation only or not
+      SIMULATE_BASELINE_ONLY = false
```

Comment:
@TMaile SIMULATE_BASELINE_ONLY, DO_MODEL_CALIBRATION , and DO_GET_RESULTS are specific to BuildingSync gem and should be defined there.  They aren't used in Extension Gem